doc/api/dgram.md

```diff
@@ -582,12 +581,11 @@ changes:
   * `type` {string} The family of socket. Must be either `'udp4'` or `'udp6'`.
     Required.
   * `reuseAddr` {boolean} When `true` [`socket.bind()`][] will reuse the
-    address, even if another process has already bound a socket on it. Optional.
+    address, even if another process has already bound a socket on it.
     Defaults to `false`.
-  * `recvBufferSize` {number} - Optional. Sets the `SO_RCVBUF` socket value.
-  * `sendBufferSize` {number} - Optional. Sets the `SO_SNDBUF` socket value.
```

Comment:
We might want to think about how to properly document this. As it is a object that is not optional but some properties are and some are not, I think it is important to know if they are required or not.

Should we add brackets around the property name to indicate this? (I know this is probably out of scope for this PR)

Reply:
I think specifying which ones are `Required" is probably enough for this, I'd assume most object properties are optional, so that can just be the default.

Thoughts?